FullGuide.md

```diff
@@ -332,15 +332,16 @@ The callback does a non-blocking read from the input stream placing the data int
                 AudioStream *oboeStream,
                 void *audioData,
                 int32_t numFrames){
-            Result result = stream2.read(audioData, numFrames, timeout);
-
-            if (result == numFrames)
-                return DataCallbackResult::Continue;
-            if (result >= 0) {
-                memset(static_cast<sample_type*>(audioData) + result * samplesPerFrame, 0,
-                    sizeof(sample_type) * (numFrames - result) * samplesPerFrame);
-                return DataCallbackResult::Continue;
-            }
+            auto result = stream2.read(audioData, numFrames, timeout);
+            if (result == Result::OK){
```

Comment:
Add a comment here why it's possible first to compare result with an value of `Result` and then with a number, and what does it mean. Otherwise might look confusing for a person not very familiar with type coercion operator overloading.

Reply:
Thanks, done.

Reply:
I don't see this update. Have you uploaded it?

Reply:
I hadn't hit save in my text editor 🤦‍♂️ . Fixed now.